MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -219,9 +276,30 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification {
   [self didReceiveUserNotification:[notification.userInfo objectForKey:NSApplicationLaunchUserNotificationKey]];
 }
 
-- (void)userNotificationCenter:(NSUserNotificationCenter *)__unused center
+- (void)userNotificationCenter:(NSUserNotificationCenter *)center
```

Comment:
As a note, we can use message forwarding to dynamically forward to the original delegate without explicitly implement all the methods. It's particularly useful for instance whenever a new method is added to the delegate, then there is no need for updating the SDK. But that can be an improvement in another story.

Reply:
+1 for doing this in a separate story.